lib/ansible/modules/network/panos/panos_pg.py

```diff
@@ -0,0 +1,233 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+#
+# Ansible module to manage PaloAltoNetworks Firewall
+# (c) 2016, techbizdev <techbizdev@paloaltonetworks.com>
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
+DOCUMENTATION = '''
+---
+module: panos_pg
+short_description: create a security profiles group
+description:
+    - Create a security profile group
+author: "Luigi Mori (@jtschichold), Ivan Bojer (@ivanbojer)"
+version_added: "2.3"
+requirements:
+    - pan-python
+options:
+    ip_address:
+        description:
+            - IP address (or hostname) of PAN-OS device
+        required: true
+    password:
+        description:
+            - password for authentication
+        required: true
+    username:
+        description:
+            - username for authentication
+        required: false
+        default: "admin"
+    pg_name:
+        description:
+            - name of the security profile group
+        required: true
+    data_filtering:
+        description:
+            - name of the data filtering profile
+        required: false
+        default: None
+    file_blocking:
+        description:
+            - name of the file blocking profile
+        required: false
+        default: None
+    spyware:
+        description:
+            - name of the spyware profile
+        required: false
+        default: None
+    url_filtering:
+        description:
+            - name of the url filtering profile
+        required: false
+        default: None
+    virus:
+        description:
+            - name of the anti-virus profile
+        required: false
+        default: None
+    vulnerability:
+        description:
+            - name of the vulnerability profile
+        required: false
+        default: None
+    wildfire:
+        description:
+            - name of the wildfire analysis profile
+        required: false
+        default: None
+    commit:
+        description:
+            - commit if changed
+        required: false
+        default: true
+'''
+
+EXAMPLES = '''
+- name: setup security profile group
+  panos_pg:
+    ip_address: "192.168.1.1"
+    password: "admin"
+    username: "admin"
+    pg_name: "pg-default"
+    virus: "default"
+    spyware: "default"
+    vulnerability: "default"
+'''
+
+RETURN = '''
```

Comment:
ok, I followed what was done in 'wakeonlan.py' given that this module has no return variables either.